scripts/export_dc2_subset.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python
+# Create a gen3 exports.yaml file containing a few filters on one patch from
+# a weekly run on /repo/dc2/.
+# Must be run on the lsst-devl cluster.
+
+import itertools
+import lsst.daf.butler
+
+patch = 24
+tract = 3828
+collection = "2.2i/runs/test-med-1/w_2022_03/DM-33223"
+
+butler = lsst.daf.butler.Butler('/repo/dc2', collections=collection)
+
+with butler.export(filename="dc2-exports.yaml") as export:
+    where = f"instrument='LSSTCam-imSim' and patch={patch} and tract={tract} and skymap='DC2'"
+    for datasetType, band in itertools.product(('deepCoadd_calexp', ), ('g', 'r', 'i')):
+        export.saveDatasets(butler.registry.queryDatasets(datasetType=datasetType,
+                                                          where=where + f" and band='{band}'"))
+
+    export.saveDatasets(butler.registry.queryDatasets(collections='skymaps',
+                                                      datasetType='skymap'))
+    export.saveDataIds(
+        butler.registry.queryDataIds(
+            ["patch"],
+            where=f"tract={tract} AND skymap='DC2'"
+        ).expanded()
+    )
```

Comment:
I think it's good practice to put the logic in a `main` function and add an `if __name__ == "__main__"` block below, even for super simple, rarely-run scripts like this.  It makes it a lot easier to upgrade it into something more while minimizing git history changes, and these scripts do end up getting copied (and cargo-culted) a fair bit.

Reply:
Well, I cargo-culted it from my own testdata_jointcal stuff, so you're absolutely right there!